### ISSUE TITLE
Code & Result on nuScenes

### ISSUE THREAD

**Issue by Treemann** [xinzhuma/monodle#17]
Hi, I have reproduced your result on KITTI. 
I can't find your submission on the nuScenes leaderboard, so I wonder if you have done experiments on the nuScenes dataset. If yes, could you share your accuracy? Will you release the code for nuScenes?

Thanks for your excellent work~

**Comment by xinzhuma** [comment]
We didn't submit our results to nuScenes leaderboard, although we did conduct some experiments on the validation set. The mAP is about 0.3, and the code will be released.

**Comment by Treemann** [comment]
When is the code expected to be released? Thanks for your sharing.

**Comment by xinzhuma** [comment]
I need some time to re-organize these codes. Unfortunately, due to the coming of CVPR/ECCV, I have no time to do this during this period. So the release of the code may take three or four months.

**Comment by Treemann** [comment]
Got it.

**Comment by Treemann** [comment]
Hi @xinzhuma , I tried to modify your code and trained the model on the nuScenes dataset. Without so much hyperparameter tuning, I got mAP=0.26 based on your model structure & training parameters for KITTI. I am tuning the parameters to see if I could get better accuracy. 
I wonder what configuration (model structure & training parameters, etc.) and tricks (like TTA, ensemble models) you use to get the result of mAP=0.3. Hope you could give some advices~

**Comment by xinzhuma** [comment]
@Treemann No TTA or model ensemble tricks. the performance of the single model can achieve about 0.3. The hyper-params are different, including the learning rate, batch size, score thresholds, etc. Besides, some instances are invisible for a specific view, and you need to remove them.

**Comment by Treemann** [comment]
@xinzhuma so the model structure is the same as that trained on KITTI ?
```
model:
  type: 'centernet3d'
  backbone: 'dla34'
  neck: 'DLAUp'
```
I'll continue to tune the hyperparameters, thanks for your reply~

**Comment by xinzhuma** [comment]
@Treemann yes, only modify the number of the channels for the heatmap branch to detect all the ten classes. we didn't predict the velocity and attributes.

**Comment by Treemann** [comment]
Hi @xinzhuma , 
I don't predict the velocity and attributes too and I get 0.276 now.
One more question, the image size of nuScenes is 1600x900 and the training is too slow, so I resize the input to 800x448. I wonder which input size & epochs you adopted.

Thanks for your patience.

**Comment by xinzhuma** [comment]
800 * 448 is okay, we also adopt this setting. You can set the confidence threshold to 0.1, which is a better setting for nuScenes. we train the models with batchsize=192 and lr=0.0005 for 140 epochs.

**Comment by Treemann** [comment]
OK, I'll try with your setting. 
(: Training for 140 epochs takes a long time compared with FCOS3D which is trained for 12 or 24 epochs~
